java/ql/lib/semmle/code/java/dataflow/internal/DataFlowUtil.qll

```diff
@@ -256,6 +274,19 @@ class MapValueContent extends Content, TMapValueContent {
   override string toString() { result = "<map.value>" }
 }
 
+/** A captured variable. */
+class ClosureContent extends Content, TClosureContent {
```

Comment:
Hmm, yes, you're right, that's probably more consistent, since the naming is "a Content that's also _prefix_" and not "a Content of a _prefix_". Will fix.